src/ui/screens/home/ViewHome.qml

```diff
@@ -21,8 +21,7 @@ MZFlickable {
     anchors.right: parent.right
 
     Component.onCompleted: {
-        Glean.sample.appStep.record({
-            action: "impression",
+        Glean.impression.mainScreen.record({
```

Comment:
Should this be "main_screen" to match the name in impression_metrics.yaml? Similar comment for the other impressions. (Perhaps I've missed something here.)

Reply:
The metric instances are generated automatically by Glean, so we can't change this. There is a little note on the different capitalization standards per language, if you are interested, on the Glean book: https://mozilla.github.io/glean/book/user/metrics/adding-new-metrics.html#capitalization tl;dr; on C++ it's camel case.